src/main/java/ExceptionHandler.java

```diff
@@ -0,0 +1,31 @@
+public class ExceptionHandler {
+    public static void checkEmptyField(String[] taskStringArr, String type) throws EmptyFieldException {
+        if (type.equals("todo") || type.equals("deadline") || type.equals("event")) {
+            if (taskStringArr.length != 2) {
+                throw new EmptyFieldException(
+                        "You're usually not that taciturn, Red. Explain things to me.");
+            } else if (!type.equals("todo")) {
+                String[] time = taskStringArr[1].split("/", 2);
+                if (time.length != 2) {
+                    throw new EmptyFieldException(
+                            "We're running short of time, so make sure you note it down.");
+                } else if (time[0].isEmpty()) {
+                    throw new EmptyFieldException(
+                            "You're usually not that taciturn, Red. Explain things to me.");
+                }
+            }
```

Comment:
Well formatted and easy to follow. However it might be difficult to extend your handler for new commands.

I believe a switch-case statement can be used here, where each case handles a single "type". If required, you can abstract out each case to avoid cluttering this method.